include/fizzy/fizzy.h

```diff
@@ -133,6 +133,32 @@ typedef struct FizzyExternalGlobal
     FizzyGlobalType type;
 } FizzyExternalGlobal;
 
+/// External kind.
+typedef enum FizzyExternalKind
+{
+    FizzyExternalKindFunction,
+    FizzyExternalKindTable,
+    FizzyExternalKindMemory,
+    FizzyExternalKindGlobal
+} FizzyExternalKind;
+
+/// Import definition.
+///
+/// @note Only one member of `desc` union corresponding to `kind` is defined for each import.
+typedef struct FizzyImportType
```

Comment:
I don't think it is type, but perhaps "description" is a better term?

Reply:
In the spec it is called just `import`, and `importdesc` is `functype | tabletype | memtype | globaltype` (i.e. everything in `import` except `module` and `name`)

Reply:
In wasm-c-api it is `importtype`, consisting of `module`, `name` and `externtype`

Reply:
Also `FizzyImportType` can be seen as consistent with other module definition types: `FizzyFunctionType`, `FizzyGlobalType` (memory and table don't define `*Type` beacuse they use `FizzyLimits` directly as a type)